dev/io.openliberty.data.internal_fat/publish/servers/io.openliberty.data.internal.fat/server.env

```diff
@@ -0,0 +1 @@
+xJAVA_HOME=/Users/njr/drivers/jdk-21.jdk/Contents/Home
```

Comment:
Looks like this was checked in accidentally since it contains a local dir

Reply:
Thanks for spotting that. I have removed it here: https://github.com/OpenLiberty/open-liberty/pull/28816/commits/bb73b3039bab99f8acfdc4b13124e51712a8581c